source/Handlebars.Benchmark/LargeArray.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using HandlebarsDotNet;
+
+namespace HandlebarsNet.Benchmark
+{
+    public class LargeArray
+    {
+        private object _data;
+        private HandlebarsTemplate<object, object> _default;
+
+        [Params(20000, 40000, 80000)]
+        public int N { get; set; }
+        
+        [GlobalSetup]
+        public void Setup()
+        {
+            const string template = @"{{#each this}}{{this}}{{/each}}";
+            _default = Handlebars.Compile(template);
```

Comment:
It's better to use `TextWriter` overload in the benchmark (like you did in `5c7fc6d27a62b21b1edb06de96e2f05844cea3e4` commit) as it allows to isolate memory allocations to internals only without noise produced by additional string allocations.

Reply:
Here you go 78dea01